obj.js

```diff
@@ -334,6 +372,15 @@ export default class BaseProtocol {
     obj[payload.name] = value;
   }
 
+  async observeListener(objDescrJSON) {
+    let obj = this.getRemoteObject(objDescrJSON.idLocal);
+    if (obj) {
```

Comment:
I don't understand this function. I don't understand why we're gating on `obj` existance, nor do I understand what else path does and (more importantly) why. Could you please help me understand? And put in code comments that allow others to understand?

Reply:
We get the first notification for the object when we subscribe to it; this happens when we first encounter the object on the server. At this point the remote stub does not yet exist on the client, since we have not yet sent the object description to the client. The subscription callback doesn't know that because the object already has a local id by this point, since we need that id in order to establish the subscription without creating a reference loop. The alternative is for the client to ignore the notification for an object it doesn't know about yet, and for the server to send all of the properties again along with the initial object description.